src/components/hmi_message_handler/src/hmi_message_handler_impl.cc

```diff
@@ -54,6 +54,7 @@ HMIMessageHandlerImpl::~HMIMessageHandlerImpl() {
   LOG4CXX_AUTO_TRACE(logger_);
   messages_to_hmi_.Shutdown();
   messages_from_hmi_.Shutdown();
+  message_adapters_.clear();
```

Comment:
@dev-gh removed in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/1be4f2b22b538d88b968d4358a69cada7af38911